docs/rfcs/active/dashboards/1778-dashboards.md

```diff
@@ -0,0 +1,117 @@
+The design should accomodate for different dashboarding ([Looker](www.looker.com), [Redash](www.redash.io)) tools used within a company.
+
+## Motivation
+
+Dashboards are a key piece within a data ecosystem of a company. They are used by different groups of employees across different organizations.
+They provide a way to visualize some data assets (tracking datasets or metrics) by allowing slice and dicing of the input data source.
+When a company scales, data assets including dashboards gets richer and bigger. Therefore, it's important to find and access to the right dashboard.
+By having dashboards as a top level entity in Datahub, we achieve below goals:
+
+- Enabling Search & Discovery for dashboard assets
+- Link dashboards to underlying data sources and have a more complete picture of data lineage
+
+## Detailed design
+
+![high level design](imgs/high_level_design.png)
+
+As shown in the above diagram, dashboards are composed of a collection of charts at a very high level. These charts
+could be shared by different dashboards. In the example sketched above, `Chart_1`, `Chart_2` and `Chart_3` are part of
+`Dashboard_A` and `Chart_3` and `Chart_4` are part of `Dashboard_B`.
+
+### Entities
+There will be 2 top level GMA [entities](../../../what/entity.md) in the design: dashboards and charts.
+It's important to make charts as a top level entity because charts could be shared between different dashboards.
+We'll need to build `Contains` relationships between Dashboard and Chart entities.
+
+### URN Representation
+We'll define two [URNs](../../../what/urn.md): `DashboardUrn` and `ChartUrn`.
+These URNs should allow for unique identification for dashboards and charts even there are multiple dashboarding tools
+are used within a company. Most of the time, dashboards & charts are given unique ids by the used dashboarding tool.
+An example Dashboard URN for Looker will look like below:
+```
+urn:li:dashboard:(Looker,<<dashboard_id>>)
+```
+An example Chart URN for Redash will look like below:
+```
+urn:li:chart:(Redash,<<chart_id>>)
+```
+
+### Chart metadata
+Dashboarding tools generally have different jargon to denote a chart.
+They are called as [Look](https://docs.looker.com/exploring-data/saving-and-editing-looks) in Looker 
+and [Visualization](https://redash.io/help/user-guide/visualizations/visualization-types) in Redash.
+But, irrespective of the name, charts are the different tiles which exists in a dashboard.
+Charts are mainly used for delivering some information in a visual way to make it easily understandable.
+They might be using single or multiple data sources and generally have an associated query running against
+the underlying data source to generate the data that it will present.
+
+Below is a list of metadata which can be associated with a chart:
+
+- Title
+- Description
+- Type (Bar chart, Pie chart, Scatter plot etc.)
+- Input sources
+- Query (and its type)
```

Comment:
Similar to Input sources(with type  of input source embedded into definition), Query should also be a explicitly defined. We dont have Query as an entity as of now, but we can define some basic structure there.

Reply:
We'll be defining a model for a `ChartQuery`. Please refer to https://github.com/linkedin/datahub/blob/a96bc672d3d6b27ee2679bfe9ea56e4d783f4753/metadata-models/src/main/pegasus/com/linkedin/chart/ChartQuery.pdl

How does its structure look like? Do you think of any additional metadata that we can capture for the query?